lib/counter_culture/configuration.rb

```diff
@@ -0,0 +1,21 @@
+module CounterCulture
+  def self.configuration
+    @configuration ||= Configuration.new
+  end
+
+  def self.configure
+    yield(configuration)
+  end
+
+  def self.reset_configuration
+    @configuration = Configuration.new
+  end
+
+  class Configuration
+    attr_accessor :use_read_replica
```

Comment:
It would be great to have this raise an exception if someone tries to set it on an older Rails version. Is that simple to add? Not a requirement if not.

Reply:
Yeah should be simple enough!

Reply:
Pushed a commit for the same